app/services/StudentsTeacher.rb

```diff
@@ -0,0 +1,8 @@
+module StudentsTeacher
+
+  def self.run(student)
+    return nil if student.classrooms.empty?
+    student.classrooms.first.teacher
```

Comment:
this should probably be `student.classrooms.last.teacher` rather than `.first` so as to return the latest teacher -- will fix